packages/eslint-plugin/lib/rules/require-img-alt.js

```diff
@@ -13,23 +13,27 @@ const { getRuleUrl } = require("./utils/rule");
 
 const MESSAGE_IDS = {
   MISSING_ALT: "missingAlt",
+  EMPTY_ALT: "emptyAlt",
+  INSERT_ALT: "insertAlt",
 };
 
 /**
  * @type {RuleModule}
  */
 module.exports = {
   meta: {
-    type: "code",
+    type: "suggestion",
 
     docs: {
-      description: "Require `alt` attribute at `<img>` tag",
+      description:
+        "Require `alt` attribute with non-empty value at `<img>` tag",
```

Comment:
I think it would be a good idea to change the description to the previous version, with empty value being valid.